hotel/models/hotel_room_type.py

```diff
@@ -8,11 +8,31 @@ class HotelRoomType(models.Model):
     _name = "hotel.room.type"
     _description = "Room Type"
 
-    name = fields.Char(required=True)
     categ_id = fields.Many2one("hotel.room.type", "Category")
     child_ids = fields.One2many(
         "hotel.room.type", "categ_id", "Child Categories"
     )
+    product_categ_id = fields.Many2one(
+        "product.category", "Product Category", delegate=True
+    )
+
+    @api.model
+    def create(self, vals):
+        if "categ_id" in vals:
```

Comment:
should be if vals.get('categ_id', False):

Reply:
Done.

Reply:
Why do you prefer `vals.get('categ_id', False)`? I feel that `if "categ_id" in vals` is more readable and _pythonesque_.